tests/ModelTest.php

```diff
@@ -1048,12 +1048,17 @@ public function testNumericFieldName(): void
 
     public function testCreateOrFirst()
     {
+        Carbon::setTestNow('2010-06-22');
+        $createdAt = Carbon::now()->getTimestamp();
```

Comment:
Consider `$currentTime` (or something more generic), since this is used for both assertions.